docs/pages/machine-id/guides/github-actions.mdx

```diff
@@ -32,116 +32,14 @@ Actions runners as well as GitHub Enterprise Server.
-- `spec.join_method` defines the join method the token is applicable for. Since this guide
-only focuses on GitHub Actions, this value should always be equal to `github`.
-- `spec.github.allow` is used to set rules for what GitHub Actions will be able to
-authenticate by using the token. In this example, the `gravitational/example`
-repository is used, and this example repository should be replaced with your own repo.
-
-Most of the configuration settings used in the example configuration file `tokenconfig.yaml`
-do not have any variable settings, however the `allow` block of `spec.github.allow` has additional
-configuration options to more strictly define which GitHub Actions tokens will be granted access to:
-
-| **Setting** | **Behavior** | **Example** |
-| ----------- | ----------- | ----------- |
-| `sub` | The subject associated with the GitHub Actions run. | `repo:example-org/example-repo:environment:development` |
-| `repository` | The repository where the action is running. | `example-org/example` |
-| `repository_owner` | The name of the organization that owns the repository where the action is running.  | `example-org` |
-| `workflow`| The name of the [workflow](https://docs.github.com/en/actions/using-workflows/about-workflows) that the running job belongs to. This references the `name` of a workflow defined within a workflow configuration file, and can be the workflow created in Step 3/3 of this guide. | `guide-demo` |
-| `environment`| The name of the [environment](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment) associated with the action run. | `development` |
-| `actor` | The GitHub account name that triggered the action run. | `username` |
-| `ref` | The git "ref" or [reference](https://docs.github.com/en/rest/git/refs#about-the-git-references-api) that triggered the action run. | `refs/heads/example` |
-| `ref_type` | The **type** of "ref" or reference that triggered the action run. | `branch` |
-
-Once the configuration YAML file has been created, apply it to your cluster using `tctl`:
-
-```code
-$ tctl create -f tokenconfig.yaml
-```
-
-The token `github-token` can now be observed with the following command, ensuring
-that the configuration is ready to be used by GitHub:
-
-```code
-$ tctl tokens ls
-```
-
-The output will resemble the following:
-
-```code
-Token       Type Labels Expiry Time (UTC)
------------ ---- ------ ----------------------------------------------
-github-token Bot         01 Jan 00 00:00 UTC (2562047h47m16.854775807s)
-```
+(!docs/pages/includes/machine-id/github-actions-create-token.mdx!)
 
 ## Step 2/3. Creating your Machine ID bot
 
 With the join token for GitHub Actions created, the next step is to create the Machine ID bot,
-and ensure that it is configured to use the newly created token. This bot can be created using a Teleport
```

Comment:
I've removed the statement regarding a `configuration.yaml` as this simply isn't true, Bots can only be created with `tctl bots add` or with Terraform at the moment.